### DISCUSSION TITLE
Will using Microsoft Aspire allow me to easily use project references during development and package references for deployment

### DISCUSSION THREAD

**Discussion by sam-wheat** [microsoft/aspire#6514] [Q&A] [answered]
I asked this question on the .net sub on Reddit - answer I got is No.  
Thought I would ask here - perhaps one of you know of a way to use the tool to do what I want. 

Using nuget packages for company-internal projects is cumbersome. In theory you should be able to develop your way up the stack and build and increment your package versions as you go. For small feature development this sometimes works, but for new development and large projects, theory and practice are quite different.

This is a problem that has plagued me for a while. Others have posted about it also. Best workaround I have found thus far is to use a project variable that if set to true means you are using project references, and when false means you are using package references. This is still klunky and time consuming.

I've been reading tidbits about Aspire and I am wondering if it can help with this problem. I don't think the product is designed to directly address this issue but I'm wondering if a more streamlined development process might be a byproduct of using it.

Would love to hear from anyone who has faced this same problem.

**Comment by davidfowl** [comment, marked as answer]
No aspire doesn’t help you do this.
